cg-spec/editors_draft.html

```diff
@@ -415,12 +415,125 @@ <h2>Concepts and Abstract Syntax</h2>
 
     <p>Again, this definition is an extension of the notion of <a>RDF dataset</a>, hence it follows that any <a>RDF dataset</a> is also an <a>RDF-star dataset</a>.</p>
 
+    <section class="informative" id="quoted-triples">
+      <h2>Quoted and Asserted Triples</h2>
+
+      <p>
+        A triple in RDF is a tuple of three components: subject, predicate, and object. These are
+        collectively known as <a>RDF Terms</a> as defined in [[RDF11-CONCEPTS]].
+      </p>
+      <p>
+        RDF-star introduces <a>quoted triple</a>, which is a new kind of RDF term.
+        A <em>quoted triple</em> is a triple used as the subject or object of another triple. Quoted
+        triples can also be called "embedded triples".
+      </p>
+
+      <p>
+        in RDF 1.1, an <a>asserted triple</a> is an element of the set of triples that
+        make up an <a>RDF graph</a>. RDF-star does not change this except that an RDF-star
+        triple can contain quoted triples. A triple can be used as an <em>asserted
+        triple</em>, a <em>quoted triple</em>, or both, in a given graph.
+      </p>
+
+      <p>
+        A quoted triple is written in Turtle-star and related syntaxes using delimiters `<<` and `>>`.
+      </p>
+      <pre data-transform="updateExample" data-content-type="text/turtle" class="nohighlight example">
+        <!--
+            << :a :name "Alice" >>
+        -->
+      </pre>  
+      <p>
+        The definition of quoted triple is recursive. A quoted triple can itself have a
+        subject or object component which is another quoted triple. Cycles of quoted
+        triples can not be created.  In this next example, there is a quoted triple with
+        property `:reportedBy` and whose subject is another quoted triple `:a :name
+        "Alice"`.
+      </p>
+      <pre data-transform="updateExample" data-content-type="text/turtle" class="nohighlight example">
+        <!--
+            << << :a :name "Alice" >> :reportedBy :charlie >>
+        -->
+      </pre> 
+    </section>
+
+    <section id="rdf-star-graph-examples">
+      <h2>RDF-star Graph Examples</h2>
+        
+      <p>
+        In the example below, there is a quoted triple, `:a :name "Alice"`, which is used
+        as the subject for an asserted triple:
+      </p>
+      <pre data-transform="updateExample" data-content-type="text/turtle" class="nohighlight example">
+        <!--
+            PREFIX : <http://www.example.org/> 
+
+            :bob :name "Bob" .
+            << :a :name "Alice" >> :statedBy :bob .
+              -->
+      </pre> 
+      <p>
+        The graph of this example has two asserted triples:
+      </p>
+      <ol>
+        <li>`:bob :name "Bob" .`</li>
+        <li>`&lt;&lt; :a :name "Alice" &gt;&gt; :statedBy :bob .`</li>
+      </ol>
+      <p>
+        This graph does not have an asserted triple, `:a :name "Alice"`, because
+        that triple is not an element of the set of triples.
+      </p>
```

Comment:
The last example of this section (below) is given without any explanation, which is strange. Here is a proposal:
```suggestion
      </p>
      <p>
        On the other hand, the graph in the example below contains three asserted triples: the same two as the previous example, and `:a :name "Alice"`. The latter is used both as an <a>asserted triple</a> and a <a>quoted triple</a> in this graph.
      </p>
```

Reply:
Done